Lib/Compiler.py

```diff
@@ -115,7 +119,7 @@ def __zip(self, staticPath, stack=""):
       if os.path.isfile(zipPath):
         if os.stat(zipPath).st_mtime > os.stat(ePath).st_mtime:
           continue
-      print "%s%s\r" % (n, " " * (20 - len(n))),
+      print("%s%s\r" % (n, " " * (20 - len(n))))
```

Comment:
The comma at the end was to avoid printing a linebreak, together with the `\r` to have some single line counter/bar/?
maybe adding keeps that?
```suggestion
      print("%s%s\r" % (n, " " * (20 - len(n))), end='')
```